src/Core/Domain/Entity/TelemetryEvent/Payload/CommandExecutionPayload.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Domain\Entity\TelemetryEvent\Payload;
```

Comment:
Could be moved one level up. Otherwise, TelemetryEventMetadata also should be extracted to a separate directory. The same for normalizers